### PR TITLE
Updated the a link to Link tag

### LINKED ISSUE #9311: [Updation] Change the a tags in the sidenav to Link tag
### Description

I found out a tags in the sideNav.js. Shouldn't it be updated to Link tag to render the specific page faster. Or it just me who wants to change it

I would like your opinion on this.

### Screenshots

Screenshot 2023-10-04 at 06 44 30

### Do you want to work on this issue?

Yes

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Shashmit**
## Fixes Issue

closes #9311 

## Changes proposed
I have changed the ```a tag``` to ```Link``` in Sidenav

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

###pre
272438217-d1c57b99-6458-4afd-967f-4627978727f8

###post 
Screenshot 2023-10-08 at 18 29 15

### CODE REVIEW

**Comment by Shashmit** [comment]
I just understood how actually the workflow manages the conflicts and errors. Sorry to the maintainer as I'm continuously  updating the branch. It wont happen happen.

**Comment by Shashmit** [comment]
@SaraJaoude can you please allow this pull request to run the workflow. I would like to see the mistakes and any other changes needed. 
Sorry to disturb you.

**Comment by ChinmayMhatre** [comment]
Hey @Shashmit , could you add some screenshot of the component. I have allowed the work flows.

**Comment by Shashmit** [comment]
yeah sure @ChinmayMhatre .
Screenshot 2023-10-10 at 21 55 06
Screenshot 2023-10-10 at 21 59 24

The change improves the full reloading to only the component that has been called when the instance has been clicked.

Screenshot 2023-10-10 at 21 59 24
The import has directly been called from the present module.

File: components/navbar/SideNav.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import ChevronRightIcon from "@heroicons/react/20/solid/ChevronRightIcon";
 import EditOnGitHub from "@components/EditOnGithub";
 import { useRouter } from "next/router";
 import { classNames } from "@services/utils/classNames";
-
+import Link from "@components/Link";
```
Comment (eddiejaoude) [inline components/navbar/SideNav.js:6]:
```suggestion
import Link from "@components/Link";

```

**Comment by eddiejaoude** [comment]
> I just understood how actually the workflow manages the conflicts and errors. Sorry to the maintainer as I'm continuously updating the branch. It wont happen happen.

you are using the default `main` branch which creates conflicts, always use a new branch created from `main`

**Review by eddiejaoude — APPROVED** [review]
Thanks for the contribution 👍

**Comment by Shashmit** [comment]
@eddiejaoude I will do as you have explained me from further on thanks for your time.

**Pull request merged**